projects/canopy/src/lib/side-nav/side-nav.model.ts

```diff
@@ -0,0 +1,4 @@
+export interface SideNavBarItem {
```

Comment:
We normally adopt the convention where the file would be `side-nav.interface.ts` and contain all the interfaces, types etc for that module.

Reply:
Fixed